### PR TITLE
Dev may2016

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Pull request for the May release.  This should only apply changes to the LEDAPS source.  The L8SR changes have been reverted back to the previous release.

### CODE REVIEW

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c [hunk 1]
```diff
@@ -4,104 +4,125 @@
...
+    t_ncep_ancillary *anc,
+    int datatype
+)
+{
+    FILE *fd = NULL;
+    char where[50],tag[50],date[50];
+    int i,grib_ret,ny,nx;
+    short year,doy,month,day,hour,minute;
+    float sec;
+    int read_grib_array(FILE *input, char *what, char *where, int *nx, int *ny,
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: I think this and the following prototype should belong in grib.h and included.  Or something similar to that.

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c [hunk 2]
```diff
@@ -112,185 +133,133 @@ int interpol_spatial_anc(t_ncep_ancillary *anc,float lat, float lon,float *value
...
+
+    return 0;
 }
 
 short getdoy(short year,short month,short day)
 {
-   int dayatmonth[]=
-       {0, 0, 31, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335, 366 };
-   int jdate;
+    int dayatmonth[]=
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: This method should probably be in date.c

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c [hunk 3]
```diff
@@ -112,185 +133,133 @@ int interpol_spatial_anc(t_ncep_ancillary *anc,float lat, float lon,float *value
...
 }
-	
+    
 int getdaymonth(short year,short doy,short *month,short *day)
 {
-	int i;
-   int daysinmonth[]=
-       {0,31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
-   short totaldays=0;
+    int i;
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: This method should probably be in date.c

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c [hunk 4]
```diff
@@ -4,104 +4,125 @@
 #include <math.h>
 #include "read_grib_tools.h"
 
-#define   LEAPYR(y) (!((y)%400) || (!((y)%4) && ((y)%100)))
+#define LEAPYR(y) (!((y)%400) || (!((y)%4) && ((y)%100)))
```
Comment (ronaldddilley) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: should be in date.h

**Comment by ronaldddilley** [comment]
No further comments

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c (hunk 1 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: Updated.

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

*NOTE – E-mail address change as of April 4, 2016.  Please update your
contact list accordingly!*

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c (hunk 2 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: Updated

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

*NOTE – E-mail address change as of April 4, 2016.  Please update your
contact list accordingly!*

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c (hunk 3 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: Updated.

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

*NOTE – E-mail address change as of April 4, 2016.  Please update your
contact list accordingly!*

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

File: ledaps/ledapsSrc/src/lndsr/read_grib_tools.c (hunk 4 above)
Comment (gschmidt-usgs) [inline ledaps/ledapsSrc/src/lndsr/read_grib_tools.c]: Updated.  MARCH definition is moved as well.

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

*NOTE – E-mail address change as of April 4, 2016.  Please update your
contact list accordingly!*

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

**Pull request closed without merging**